### DISCUSSION TITLE
Blazorise should provide Tailwind CSS as option

### DISCUSSION THREAD

**Discussion by BraveEvidence** [Megabit/Blazorise#1590] [General]
Blazorise should provide Tailwind CSS as option

**Comment by stsrki** [comment]
Tailwind is already planned https://github.com/stsrki/Blazorise/issues/825

It's currently not a priority since we're more focused on cleaning the API and making Blazorise more stable. Creating a new provider takes time and so it will probably be done after the 1.0 milestone.

**Comment by tesar-tech** [comment]
This may be useful for creating ui with tailwind since you already have the bootstrap one: https://awssat.com/opensource/tailwindo/2_quick-start.html#convert-a-whole-directory

**Reply by stsrki** [reply]
I will definitely have a look. Thanks

**Comment by harperjohn** [comment]
Is Tailwind support still planned?

**Reply by stsrki** [reply]
Absolutely. But not until we release v1.0, which is planned for Feb 2022. We want to first stabilize all the APIs before committing to larger features.

**Comment by buvinghausen** [comment]
@stsrki I saw you merged the Tailwind provider in ~3 weeks ago so A) thank you! and B) do you know when it will make it out to NuGet?  Is there a private feed server I can pull it off of in the meantime?  I want to migrate to Blazorise and would like to use Tailwind over the existing CSS options.

**Reply by stsrki** [reply]
I will release a preview on NuGet today, and you can try it. Once released it would be great if you could give us any feedback so that we can try to fix any issues before the final release 😊

**Reply by stsrki** [reply]
@buvinghausen Preview1 is now available from NuGet.

**Reply by buvinghausen** [reply]
> @buvinghausen Preview1 is now available from NuGet.

Cool I will start banging on it this weekend. I appreciate the speedy turnaround!

**Reply by stsrki** [reply]
> > @buvinghausen Preview1 is now available from NuGet.
> 
> Cool I will start banging on it this weekend. I appreciate the speedy turnaround!

FYI, the preview documentation is also up https://preview.blazorise.com/
